modules/consul-cluster/variables.tf

```diff
@@ -3,10 +3,19 @@
 # You must provide a value for each of these parameters.
 # ---------------------------------------------------------------------------------------------------------------------
 
+variable "project" {
+  description = "The project to deploy the cluster in"
+}
+
 variable "gcp_region" {
   description = "All GCP resources will be launched in this Region."
 }
 
+variable "gcp_zones" {
+  description = "All zones you want to deploy to within the specified region"
+  type = "list"
+}
+
```

Comment:
Why add this var? It doesn't appear to be used anywhere?

Reply:
Removed in 19dd4d2